util/pingpong.c

```diff
@@ -1483,8 +1483,8 @@ static int pp_init_ep(struct ct_pingpong *ct)
 		return ret;
 	}
 
-	ret = pp_post_rx(ct, ct->ep, MAX(ct->rx_size, PP_MAX_CTRL_MSG),
-			 ct->rx_ctx_ptr);
+	ret = pp_post_rx(ct, ct->ep, MAX(ct->rx_size, PP_MAX_CTRL_MSG) +
+			 ct->msg_prefix_size, ct->rx_ctx_ptr);
```

Comment:
The code for handling msg_prefix_size needs other cleanup.  Sometimes it is added in pp_post_rx, but not always.  And pp_post_tx never adds it.  There needs to be some consistency made throughout the code regarding where it is added.

Reply:
I can believe that this change works.  The existing code (prior to this patch) just isn't very clean.  For example, pp_post_rx also adds the msg_prefix_size and calls MAX() again.

Reply:
Sure, I'll examine the existing code and fix the problems within this patch

Reply:
I will take a shot at cleaning this up and open a new PR once done.  I won't be able to properly test it, however.